css/style.css

```diff
@@ -0,0 +1,16 @@
+@media (min-width: 1281px) {
+#navbar{
+    width: 20%;
+    background-color: gray;
+    height: 100%;
+    background-color: white;
+}
+#main-doc{
+    width: 80%;
+}
+
+nav-link {
+    height: 30px;
+    line-height: 30px;
+}
```

Comment:
Kindly notice that this selector has a . missing on its start, please adjust that 😎 .